### PR TITLE
Setup travis

### PR DESCRIPTION
**Pull request by xschildw**
Can we try this? Just want to see if it's going to at least install the packages.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+r_packages:
+  - readxl
+  - rjson
+  - stringr
+  - org.Hs.eg.db
+  - xlsx
+  - tm
+  - slam
+before_install:
+  - cd code
```
Comment (zaro0508) [inline .travis.yml]: this should probably be under `script:` section

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+language: r
+cache: packages
+sudo: false
```
Comment (zaro0508) [inline .travis.yml]: should add `fast_finish: true`

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+language: r
+cache: packages
```
Comment (zaro0508) [inline .travis.yml]: travis doc says it should be this?
```
cache:
  packages: true
```

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  - rjson
+  - stringr
+  - org.Hs.eg.db
+  - xlsx
+  - tm
+  - slam
+before_install:
+  - cd code
+script:
+  - Rscript ./AACR2018.R
```
Comment (zaro0508) [inline .travis.yml]: this build runs in the data folder?  will generated files go into the correct directory?

File: .travis.yml (hunk 1 above)
Comment (xschildw) [reply .travis.yml]: OK, between before_install, after_success, script... not always clear what goes where...

File: .travis.yml (hunk 2 above)
Comment (xschildw) [reply .travis.yml]: OK

File: .travis.yml (hunk 3 above)
Comment (xschildw) [reply .travis.yml]: Don't think so, see https://docs.travis-ci.com/user/caching/#r-package-cache

File: .travis.yml (hunk 4 above)
Comment (xschildw) [reply .travis.yml]: It run in the code folder, looking at the script it refers to the data as '../data/xxx.yyy'.

**Comment by xschildw** [comment]
Any way to test this locally before submitting a pull request (even a linter would help)?
Looks like I'm going to have to use a matrix setup, the language directive only supports
one language and I need python to deploy. Correct?

File: .travis.yml (hunk 3 above)
Comment (zaro0508) [reply .travis.yml]: ahh ok. looks good then.  i should have linked the example i saw, it was here: https://docs.travis-ci.com/user/languages/r/#packrat

File: .travis.yml (hunk 4 above)
Comment (zaro0508) [reply .travis.yml]: ok.

**Pull request merged**